_layouts/default.html

```diff
@@ -26,23 +31,13 @@
 
     <aside>
       <h1 class="page-title"><a href="index.html">Qu Docs</a></h1>
-      <p class="intro">Stand up your own data API using our open-source platform.</p>
-      <nav>
-        <ul>
-          <li><a href="index.html">About Qu</a></li>
-          <li><a href="getting-started.html">Getting Started</a></li>
-          <li><a href="endpoints.html">API Endpoints</a></li>
-          <li><a href="queries.html">Query API</a></li>
-          <li><a href="data_formats.html">Data Formats</a></li>
-          <li><a href="data_publishing.html">Publishing Data with Qu</a></li>
-          <li><a href="roadmap.html">Roadmap</a></li>
-          <li><a href="https://github.com/cfpb/qu">Code on Github</a></li>
-        </ul>
-      </nav>
     </aside>
 
     <section class="main-content" role="main">
-      {{ content }}
+      <p>
+        This page has moved to
```

Comment:
@niqjohnson Do you think this should say something like "This page is archived and longer active. It has moved to..." ?

Reply:
@marcesher: In theory no one will ever see that text (or will see it only for a blink of an eye) as the meta refresh should instantly kick you over to the Qu repo's landing page. It's there just in case something goes wrong with that redirection, and I followed [the wording we used when we shut down the Design Manual](https://github.com/cfpb/design-manual/blob/gh-pages/index.html#L12) a couple of months ago. I don't have any preferences either way; I'm happy to change this to what you suggested if that will better match what you'll be adding to the Qu README.

Reply:
Roger all that. I see now what you're saying. Thanks!